test-automation-school/src/main/java/test/automation/school/pageobjects/SearchPage.java

```diff
@@ -0,0 +1,60 @@
+package test.automation.school.pageobjects;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * Created by Max on 5/17/2017.
+ */
+public class SearchPage {
+    private WebDriver driver;
+    private WebDriverWait wait;
+
+    @FindBy(xpath = "//*[@name='btn'] [@value='Искать' or @value='Meklēt']")
+    @CacheLookup
+    public WebElement searchBtn;
+
+    @FindBy(xpath = "//input[@id='ptxt']")
```

Comment:
why use XPATH in this case, if there`s specially designated locator - _By.id_. Moreover, no need to specify the tag here ("input"), since "id" attribute ensures that no other element with such id won`t be present on the page